### PR TITLE
[PT-BR] New sentences and some non-translated sentences translations …

### PR DESCRIPTION
**Pull request by gohlas**
Added various updates to Pt-BR intents.

---------

### CODE REVIEW

File: tests/pt-br/cover_HassTurnOff.yaml [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+  - sentences:
+      - fechar a porta da garagem
+    intent:
+      name: HassTurnOff
+      slots:
+        device_class: garage
+        domain: cover
+    response: Fechado
+  - sentences:
+      - fechar as cortinas no cozinha
```
Comment (luyzfernando08) [inline tests/pt-br/cover_HassTurnOff.yaml:12]: Could you adjust the sentence to "Fechar as cortinas da cozinha"

File: tests/pt-br/cover_HassTurnOn.yaml [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+    intent:
+      name: HassTurnOn
+      slots:
+        device_class: garage
+        domain: cover
+    response: Aberto
+
+  - sentences:
+      - abra as cortinas no cozinha
+      - abrir as cortinas no cozinha
```
Comment (luyzfernando08) [inline tests/pt-br/cover_HassTurnOn.yaml:15]: Could you adjust the sentence to "abrir as cortinas da cozinha"

File: tests/pt-br/cover_HassTurnOn.yaml [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+      - abrir a porta da garagem
+    intent:
+      name: HassTurnOn
+      slots:
+        device_class: garage
+        domain: cover
+    response: Aberto
+
+  - sentences:
+      - abra as cortinas no cozinha
```
Comment (luyzfernando08) [inline tests/pt-br/cover_HassTurnOn.yaml:14]: Could you adjust the sentence to "abra as cortinas da cozinha"

File: tests/pt-br/light_HassLightSet.yaml [hunk 4]
```diff
@@ -1,6 +1,65 @@
...
+      name: HassLightSet
+      slots:
+        brightness: 50
+        area: Quarto
+        name: all
+    response: Brilho definido
+
+  # color
+  - sentences:
+      - põe a luz do quarto a vermelho
```
Comment (luyzfernando08) [inline tests/pt-br/light_HassLightSet.yaml:33]: Here we can change it to "para vermelho", what do you think?

File: tests/pt-br/light_HassLightSet.yaml [hunk 5]
```diff
@@ -1,6 +1,65 @@
...
+      - põe a luz do quarto vermelha
+      - põe a cor da luz do quarto a vermelho
```
Comment (luyzfernando08) [inline tests/pt-br/light_HassLightSet.yaml:35]: same thing here

File: tests/pt-br/_fixtures.yaml
Comment (luyzfernando08) [inline tests/pt-br/_fixtures.yaml:1]: According to the [documentation](https://developers.home-assistant.io/docs/voice/intent-recognition/test-syntax), we should not use generic names in tests.

Could you correct the items related to the "Ventilador"?

**Review by luyzfernando08 — COMMENTED** [review]
Hello @gohlas, I made some comments. 

Feel free to question any comments.

Thanks.

File: tests/pt-br/cover_HassTurnOff.yaml (hunk 1 above)
Comment (gohlas) [reply tests/pt-br/cover_HassTurnOff.yaml]: ok.

File: tests/pt-br/cover_HassTurnOn.yaml (hunk 2 above)
Comment (gohlas) [reply tests/pt-br/cover_HassTurnOn.yaml]: ok

File: tests/pt-br/cover_HassTurnOn.yaml (hunk 3 above)
Comment (gohlas) [reply tests/pt-br/cover_HassTurnOn.yaml]: ok

File: tests/pt-br/light_HassLightSet.yaml (hunk 4 above)
Comment (gohlas) [reply tests/pt-br/light_HassLightSet.yaml:36]: Ok Here, but there is a lot of sentences from pt from Portugal,
I will make another PR later on to correcet these senteces.
Many of them are not really used in pt-br.
If we agree we can keep like that for now and update this later.

File: tests/pt-br/light_HassLightSet.yaml (hunk 5 above)
Comment (gohlas) [reply tests/pt-br/light_HassLightSet.yaml:38]: same as above.

File: tests/pt-br/_fixtures.yaml
Comment (gohlas) [reply tests/pt-br/_fixtures.yaml:1]: ok

**Comment by gohlas** [comment]
@luyzfernando08 
Done, see the comments.

**Review by luyzfernando08 — APPROVED** [review]

**Comment by gohlas** [comment]
Hi @luyzfernando08
I just fixed the conflict with you PR #1967 .
Maybe you will need to aprove this again?

**Pull request merged**